src/win32/ReadLoopRunner.cpp

```diff
@@ -204,8 +214,10 @@ void ReadLoopRunner::read() {
     &ReadLoopRunner::eventCallback
   )) {
     setError("Service shutdown unexpectedly");
```

Comment:
In another PR we should revisit the use of setError here for ERROR_OPERATION_ABORTED.